gluten-core/src/main/scala/io/glutenproject/execution/SortExecTransformer.scala

```diff
@@ -18,54 +18,82 @@
 package io.glutenproject.execution
 
 import java.util
-
```

Comment:
Just a tiny code format issue. Let's keep using empty line to separate different dependency groups.

Reply:
Thanks for the comment. Fixed.